src/features/timeline/LastPeriodAsk.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from "react"
+import { Box, Button } from "@material-ui/core"
+import { useDispatch, useSelector } from "react-redux"
+import { upsertEntry } from "../entries"
+import LastDateInput from "../onboarding/Onboarding/LastDateInput"
+import { selectMainMensesTag } from "../settings"
+
+const LastPeriodAsk = () => {
+  const dispatch = useDispatch()
+  const [lastPeriod, setLastPeriod] = useState(new Date())
+  const mainMensesTag = useSelector(selectMainMensesTag)
+
+  const submitPeriodDate = async () => {
+    try {
+      await dispatch(
+        upsertEntry(lastPeriod, {
+          note: `#${mainMensesTag}`,
+        })
+      )
+    } catch (e) {
+      // install a toast library?
```

Comment:
Could take this up next, it would be nice to always have something on errors

Reply:
Yeah that could be nice, I think there is one that is recommended by MaterialUI...

Reply:
yeah its here: https://material-ui.com/components/snackbars/

Will implement it :)

Reply:
Implemented